### PR TITLE
Update config-settings.rst

### PR DESCRIPTION
**Pull request by amyblais**
Please review specific sections and/or provide correct content for each new config setting:

@lieut-data ``Require Plugin Signature`` and ``SignaturePublicKeyFiles``.
@wiersgallak or @thefactremains All the other settings.

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -2054,6 +2054,14 @@ This filter is defaulted to ``(|(objectClass=group)(objectClass=groupOfNames)(ob
 | This feature's ``config.json`` setting is ``"GroupFilter": ""`` with string input.                                                                                   |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Guest Filter
+^^^^^^^^^^^^^^
```
Comment (wiersgallak) [inline source/administration/config-settings.rst]: @amyblais Justine has these documented in this open PR: https://github.com/mattermost/docs/pull/3213

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -2284,6 +2284,27 @@ Enter ``https://<your-mattermost-url>/login/sso/saml`` (example: ``https://examp
...
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+CanonicalAlgorithm
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"CanonicalAlgorithm": ""`` with string input.                                                                   |
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+GuestAttribute
```
Comment (wiersgallak) [inline source/administration/config-settings.rst]: This one is also in Justine's pull request - I missed it yesterday.

File: source/administration/config-settings.rst [hunk 3]
```diff
@@ -2683,6 +2699,28 @@ Lists installed plugins on your Mattermost server and whether they are enabled.
...
 
+Require Plugin Signature
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**True**:
+
+**False**:
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"RequirePluginSignature": {}`` with object input mapping plugin IDs as keys to objects, each of which contains a key ``"Enable": false`` with options ``true`` or ``false``. |
```
Comment (lieut-data) [inline source/administration/config-settings.rst]:
```suggestion
| This feature's ``config.json`` setting is ``"RequirePluginSignature": true`` with options ``true`` and ``false``.  |
```

File: source/administration/config-settings.rst [hunk 4]
```diff
@@ -2683,6 +2699,28 @@ Lists installed plugins on your Mattermost server and whether they are enabled.
 | This feature's ``config.json`` setting is ``"PluginStates": {}`` with object input mapping plugin IDs as keys to objects, each of which contains a key ``"Enable": false`` with options ``true`` or ``false``. |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Require Plugin Signature
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**True**:
```
Comment (lieut-data) [inline source/administration/config-settings.rst]:
```suggestion
**True**: Require valid plugin signatures before starting managed or unmanaged plugins. Prepackaged plugins are not subject to plugin signature verification. Pre-packaged plugins are not subject to plugin signature verification. Plugins installed through the Plugin Marketplace are always subject to plugin signature verification at the time of download.
```

File: source/administration/config-settings.rst [hunk 5]
```diff
@@ -2683,6 +2699,28 @@ Lists installed plugins on your Mattermost server and whether they are enabled.
...
+
+**False**:
```
Comment (lieut-data) [inline source/administration/config-settings.rst]:
```suggestion
**False**: Do not require valid plugin signatures before starting managed or unmanaged plugins. Pre-packaged plugins are not subject to plugin signature verification. Plugins installed through the Plugin Marketplace are always subject to plugin signature verification at the time of download.
```

File: source/administration/config-settings.rst [hunk 6]
```diff
@@ -2683,6 +2699,28 @@ Lists installed plugins on your Mattermost server and whether they are enabled.
...
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"RequirePluginSignature": {}`` with object input mapping plugin IDs as keys to objects, each of which contains a key ``"Enable": false`` with options ``true`` or ``false``. |
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Signature Public Key Files
```
Comment (lieut-data) [inline source/administration/config-settings.rst]: This is actually an array field, whose contents are relative or absolute paths to signature files. In addition to the Mattermost plugin signing key built-into the server, each public key specified here is trusted to validate plugin signatures.

File: source/administration/config-settings.rst [hunk 7]
```diff
@@ -2702,23 +2702,21 @@ Lists installed plugins on your Mattermost server and whether they are enabled.
 Require Plugin Signature
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-**True**:
+**True**: Require valid plugin signatures before starting managed or unmanaged plugins. Prepackaged plugins are not subject to plugin signature verification. Pre-packaged plugins are not subject to plugin signature verification. Plugins installed through the Plugin Marketplace are always subject to plugin signature verification at the time of download.
```
Comment (lieut-data) [inline source/administration/config-settings.rst]: Looks like `Pre-packaged plugins are not subject to plugin signature verification.` got duplicated twice -- sorry for that!

**Pull request merged**